roles/endpoints/defaults/main.yml

```diff
@@ -3,6 +3,8 @@ endpoints:
   main: "{{ fqdn }}"
   db: "{{ undercloud_floating_ip }}"
   rabbit: "{{ undercloud_floating_ip }}"
+  mongodb: "{{ undercloud_floating_ip }}"
+  magnum: "{{ fqdn }}"
```

Comment:
These were going to be added as future work I thought. Did you need them here? We deployed mongo/magnum?